test/test_secure_ros/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(test_secure_ros)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  std_msgs
+)
+
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES test_secure_ros
+   CATKIN_DEPENDS roscpp std_msgs
+#  DEPENDS system_lib
+)
+
+include_directories( ${catkin_INCLUDE_DIRS} )
```

Comment:
consider target_include_directories to prevent "leaking" include directories to other targets